### PR TITLE
[BI-997 BI-1061]-2 Update Trait Import (clean redo)

### PR DESCRIPTION
**Pull request by HMS17**
Ontology Changes 
- Replace "Trait" with "Ontology" in Trait Import
- Alter table columns in Trait Import according to specifications
- Changed "traits" to "ontology terms" in modal message and banner messages

Linked to PR [taf/BI-997](https://github.com/Breeding-Insight/taf/pull/21) 
Redo of PR [bi-web/BI-997](https://github.com/Breeding-Insight/bi-web/pull/116)

### CODE REVIEW

**Review by nickpalladino — COMMENTED** [review]
Looks like some of the table unit tests are failing, you figuring it out?

**Comment by HMS17** [comment]
> Looks like some of the table unit tests are failing, you figuring it out?

I'm looking into it. For some reason the name column isn't retrieving the name....or any test value I put in there.

**Comment by HMS17** [comment]
> > Looks like some of the table unit tests are failing, you figuring it out?
> 
> I'm looking into it. For some reason the name column isn't retrieving the name

Thanks for the help in fixing the tests @nickpalladino !

**Review by nickpalladino — APPROVED** [review]

File: src/views/trait/TraitsImport.vue [hunk 1]
```diff
@@ -46,9 +46,9 @@
     </WarningModal>
 
     <template v-if="state === ImportState.CHOOSE_FILE || state === ImportState.FILE_CHOSEN">
-      <h1 class="title" v-if="showTitle">Import Traits</h1>
-      <ImportInfoTemplateMessageBox v-bind:import-type-name="'Trait'"
-                                    v-bind:template-url="'https://cornell.box.com/shared/static/u9vuob3vvu2cgwyoe26b2722u41b8f6o.xls'"
+      <h1 class="title" v-if="showTitle">Import Ontology</h1>
+      <ImportInfoTemplateMessageBox v-bind:import-type-name="'Ontology'"
+                                    v-bind:template-url="'https://cornell.box.com/shared/static/8sp0qvccpjotosiv8576tczeg09nnvao.xls'"
```
Comment (timparsons) [inline src/views/trait/TraitsImport.vue:51]: This template link is to v9.  The previous link is to v10.  I think we want v10 as the link, @dmeidlin can you verify?

**Review by timparsons — CHANGES_REQUESTED** [review]
Similar to a comment Nick left in the original pull request, in `TraitUploadService.ts`, the constant `forbiddenUploadingFile` references "trait", and should probably be updated to "ontology term"

**Comment by HMS17** [comment]
> Similar to a comment Nick left in the original pull request, in `TraitUploadService.ts`, the constant `forbiddenUploadingFile` references "trait", and should probably be updated to "ontology term"

Added changes.

File: src/views/trait/TraitsImport.vue (hunk 1 above)
Comment (dmeidlin) [reply src/views/trait/TraitsImport.vue:51]: v10 has the new required fields for the ontology mvp, so, yes, keep the previous link.

Comment (HMS17) [reply src/views/trait/TraitsImport.vue]: Rectified

**Review by timparsons — APPROVED** [review]

**Pull request merged**